src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,35 @@
 package mate.jdbc;
 
+import mate.jdbc.dao.ManufacturerDao;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.util.ConnectionUtil;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
+        ConnectionUtil.getConnection();
```

Comment:
```suggestion
        ConnectionUtil.getConnection();
```
Is the Main class a place for this?